app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -17,27 +17,25 @@ import {
   Select,
 } from 'tno-core';
 
-import { IDateOptions, IPressMember } from './interfaces';
+import { IDateOptions, IGroupedDates, IPressMember } from './interfaces';
 import * as styled from './styled';
-import { generateDates } from './utils';
+import { generateDates, seperateAlias } from './utils';
 
 export const PressGallery: React.FC = () => {
+  const [, api] = useContributors();
   const [
-    {
-      home: { filter },
-    },
-    { findContentWithElasticsearch },
+    { pressGalleryFilter },
+    { storeGalleryDateFilter, findContentWithElasticsearch, storeGalleryPressFilter },
   ] = useContent();
-  const [, api] = useContributors();
-  const [{ pressGalleryFilter }, { storeGalleryDateFilter, storeGalleryPressFilter }] =
-    useContent();
 
   const [content, setContent] = React.useState<IContentSearchResult[]>([]);
   const [pressMembers, setPressMembers] = React.useState<IPressMember[]>([]);
+  const [initialLoad, setInitialLoad] = React.useState(false);
```

Comment:
does this mean "is initial load underway"?

Reply:
correct